packages/core/src/Constants/contract.js

```diff
@@ -98,22 +98,6 @@ export const getMarketNamesMap = () => ({
 });
 
 export const getUnsupportedContracts = () => ({
-    EXPIRYMISS: {
```

Comment:
It looks like a duplicated file, can we remove it completely?

Reply:
Removing one of the files helps with the high duplication percentage on the PR 👌🏼

Reply:
There are almost similar, but files, which use those functions, import them from different packages. For e.g. in _packages/trader/src/Modules/Trading/Containers/purchase.jsx_ file `getContractTypePosition` (which calls inside `getUnsupportedContracts`) is imported from `trader package` and _packages/components/src/components/positions-drawer-card/positions-drawer-card.jsx_ file import `getContractTypeDisplay`(which also calls inside `getUnsupportedContracts`) from `shared package`.

Reply:
I think we should import from shared package. I can compare those similar files, add the differences in contract,js in @deriv/shared and change all imports in order to refer to one file. But there are a lot of files to change, so is it better to do it in this PR or create a card for improvement?